ansible/roles/base/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+
+---
+
+    - name: Update apt cache
+      apt: update-cache=yes cache_valid_time=86400
+      become: yes
+      become_method: sudo
```

Comment:
You don't need the become: yes and become_method here as you are setting them in vagrant.yml